### DISCUSSION TITLE
Contribution Question: Browser Support / Polyfills

### DISCUSSION THREAD

**Discussion by illusionalsagacity** [TheSpyder/rescript-webapi#24] [Q&A] [answered]
Is there an expectation of what browser or spec level to bind against? i.e. how much should these bindings take on for ensuring functionality vs being dependent on external build tools to polyfill?

For example, if writing bindings for a newer feature, technically you could bind to the polyfill instead of expecting the bound api to be there.

**Comment by TheSpyder** [comment, marked as answer]
I don't want to bundle any polyfills. I am happy to accept bindings for things not supported by all browsers, but I’d prefer to stick to accepted standards unless there’s a good reason to add a bleeding edge api.

This webapi should be similar to using browsers directly - if a target browser doesn’t support a particular feature a polyfill must be added to the project. This way projects can choose their own polyfills instead of having it be dictated by this webapi.

We could perhaps offer bindings to aid in feature detection, but that’s as far as it should go.
